src/platforms/mirserver/wayland-generated/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+set(PROTOCOL "qt-windowmanager")
```

Comment:
Since we seem to autogenerate the wrapper files why do we include them in source also?

Reply:
I'm not sure. It's what Alan did.